### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by Oleksii14** [vuejs/pinia#1767] [Help and Questions] [answered]
Hello! Is there a way to contribute without being a team member? I cloned a repo and added a feature I would like to share, but I cannot push a branch and create a PR. Thanks!

**Comment by rustamd** [comment, marked as answer]
You can create PR from forked repo. 

<details>
  Доброго вечора, ми з України!
</details>

**Reply by Oleksii14** [reply]
So I have to fork a repo and then create a PR there?

**Reply by posva** [reply]
yes
